src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+/**
+ * DukeException represents the possible exceptions that might occur while using Duke.
+ */
+public class DukeException extends RuntimeException {
+    private final String message;
+
+    /**
+     * Constructs a DukeException
```

Comment:
Should there be a punctuation after the Javadoc header description?

```suggestion
     * Constructs a DukeException.
 ```